tutorials/nservicebus-101/index.md

```diff
@@ -4,32 +4,20 @@ suppressRelated: true
 reviewed: 2017-01-26
 ---
 
-In this course, you will learn how to get working quickly with NServiceBus.
+partial: nsb101-intro-paragraph
 
+In [Lesson 1: Getting started](lesson-1/) (10-15 minutes) you will learn how to set up your development environment and create your very first messaging endpoint.
 
-## Objectives
+In [Lesson 2: Sending a command](lesson-2/) (15-20 minutes) you will learn how to define messages and message handlers, and send your first message.
 
-In this course we'll build a messaging system for a retail e-commerce domain.
+In [Lesson 3: Multiple endpoints](lesson-3/) (15-20 minutes) you will learn how to create multiple endpoints and send messages between them.
 
-By the end of the course, you will have learned:
+In [Lesson 4: Publishing events](lesson-4/) (25-30 minutes) you will learn how to publish events to multiple subscribers, and about the benefits of decoupling business processes using the Publish/Subscribe pattern.
```

Comment:
>learn how to publish events to multiple subscribers

does the user know pub/sub yet? This would be better as: how to broadcast a message from one endpoint to multiple endpoints using the Publish/Subscribe messaging pattern.